doc/news/_preparation_next_release.md

```diff
@@ -338,7 +338,7 @@ you up to date with the multi-language support provided by Elektra.
 The website is generated from the repository, so all information about
 plugins, bindings and tools are always up to date. Furthermore, we changed:
 
-- <<TODO>>
+- Added github build status badges to website @hesirui
```

Comment:
You need to surround your name with `_()_`, i.e. change `(hesirui)` to `_(hesirui)_`. See also what the build server says: https://build.libelektra.org/jenkins/blue/organizations/jenkins/libelektra/detail/PR-2546/3/